src/redux/products/products.reducer.js

```diff
@@ -38,13 +38,162 @@ export const initialState = {
   filters: {
     colorsFilter: [],
     patternsFilter: [],
-    categoryFilter: null,
+    categoryFilter: [],
     priceFilter: [0, 99999999],
     searchFilter: '',
     modelsFilter: [],
     isHotItemFilter: false
   },
-  filterData: [],
+  filterData: {
+    categories: [
```

Comment:
I think it would be better to set empty array

Reply:
fixed